app/views/homepage/_grid_item.haml

```diff
@@ -19,7 +19,8 @@
           .home-fluid-thumbnail-grid-details
             = link_to(person_path(id: listing.author.username), class: "home-fluid-thumbnail-grid-details-author-name", title: name) do
               = name
-            - dist = number_with_delimiter(distance.round(2), locale: locale)
+            - dist_w_decimal_precision = (distance < 1) ? number_with_precision(distance, precision: 1, significant: true, locale: locale) : number_with_precision(distance, precision: 2, significant: true, locale: locale)
```

Comment:
I'll change this:
precision = (distance < 1) ? 1 : 2
dist = (distance < 0.1) ? "< 0.1" : number_with_precision(distance, precision: precision, significant: true, locale: locale)